src/Api/Controllers/OrganizationsController.cs

```diff
@@ -220,6 +220,7 @@ public async Task<OrganizationResponseModel> Post([FromBody] OrganizationCreateR
         return new OrganizationResponseModel(result.Item1);
     }
 
+    // TODO: move to SelfHostedOrganizationLicensesController
```

Comment:
I haven't done this yet because I think there's already quite a bit in this PR. Reviewer: let me know if you agree and if so, whether you want it in this PR or a follow-up PR.

Reply:
I say go ahead and move these over to the `SelfHostedOrganizationLicensesController`, I think this PR has the capacity.

Reply:
Done, note that the routes are also tweaked/changed slightly now that they're sitting under the `organizations/licenses/self-hosted` route already.